### PR TITLE
Conduct: Add H1 2022 Transparency Report

### PR DESCRIPTION
**Pull request by salaxander**
Add the first transparency report for 2022 to the conduct directory. Also including a small update to the readme to reflect membership

**Which issue(s) this PR fixes**:

Fixes #

### CODE REVIEW

**Review by endocrimes — APPROVED** [review]
/approve
/lgtm
/hold
^ for a second review

**Review by jeremyrickard — APPROVED** [review]
/lgtm

**Pull request merged**
